docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,255 @@
+Kubernetes.  The steps would be something like this:
+
+1. Identify the required values for connecting the application to the database
+2. Locate the resources where the values are present
+3. Take the values from different resources and create a Secret resource
+4. Mount the Secret resource into the application
+5. Depending on the application requirement the values should be exposed as env var or file.
+
+In this quick start, we are going to leverage Service Binding as a way to easily
+and safely connect the application to the database service.  In order to do
+that, we'll need to create a Service Binding ressource which will trigger the
+Service Binding Operator to inject the binding metadatas into the application.
+
+
+1. Create the ServiceBinding custom resource to inject the bindings:
+
+The `.spec` has two sections, the first one is a list of service resurces
+(`.spec.services`) and the second one is the `.spec.application`.  The services
+resources points to backing service resources.  How the values are exposed from
+service resources are explained [Exposing binding
+data](../exposing-binding-data/intro-expose-binding) section.  In the below
+ServiceBinding resource, there are two service resources, one custom resource
+and another Secret resource.  This is required as the values required for
+database connectivity is distributed in these two resources.  The application
+points to a `Deployment` or any similar resource with an embedded `PodSpec`.
+The mappings has extra mappings required for connectity.
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: my-postgresql
+spec:
+    services:
+    - group: "crunchydata.com"
+      version: v1
+      kind: Pgcluster
+      name: hippo
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-hippo-secret
+    application:
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+    mappings:
+    - name: type
+      value: "postgresql"
+```
+
+For simplicity, you can copy/paste the following command to create the resource:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/service-binding.yaml
+```
+
+To learn more about creating service bindings, you can find more information on
+the following
+[document](../creating-service-bindings/creating-service-binding)..
+
+The values from backing service resource is injected into the application
+container as files.  It will be injected under `/bindings/spring-petclinic-rest`
+directory.  All the values from the Secret resource is injected.  In the above
+example, that is `username` and `password`.  And the values pointed out through
+the annotation are also injected which includes `database`, `host`, and
+`password`.  Finally, from the mappings, `type` is also injected.  The
```

Comment:
why password is injected two times?

Reply:
Corrected. It's `port`. dec29d0c9c6707818f222bcda65b16d3bda5a3fc